apps/backend/tsconfig.json

```diff
@@ -5,6 +5,7 @@
     "declaration": true,
     "removeComments": true,
     "emitDecoratorMetadata": true,
+    "noImplicitAny": true,
```

Comment:
This can be set in the root `tsconfig.json` and removed from both the backend and frontend `tsconfig.json`.